README.md

```diff
@@ -74,25 +74,24 @@ If you already have a model API endpoint available you can skip this process.
 
 ### Deploy the Web App
 
-1. Press the ``Deploy to IBM Cloud`` button. If you do not have an IBM Cloud account yet, you will need to create one.
+1. Press the `Deploy to IBM Cloud` button. If you do not have an IBM Cloud account yet, you will need to create one.
 
     [![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/MAX-Image-Caption-Generator-Web-App)
 
-2. Click `Delivery Pipeline` and fill in the form with Image Caption Generator API endpoint deployed above then
-click on ``Create``.
+2. Click `Delivery Pipeline` and fill in the form with the Image Caption Generator API endpoint deployed above, then click on `Create`.
 
-    > e.g. `https://max-image-caption-generator.mybluemix.net/model/predict`
+    > The format for this entry should be `<Public IP>:<NodePort>`
```

Comment:
You are correct that this should change since we found that you can't get domain names for container services, but this example is good. the endpoint needs to include the `http://` so I think a better note here would be:

```
e.g. https://170.0.0.1:5000/model/predict
```

this would get across that a ip and port are needed with an obviously useless ip

Reply:
@djalova opinion?

Also we'll need to update the deploy and toolchain files as well since this url came from there.

Reply:
I think this makes sense, but adding the `/model/predict` at the end seems unnecessary if that part is added in the code.

Reply:
fair enough I'll change it to that and merge this, I'll also update the other files